features/buyer/tell_us_about_contract.feature

```diff
@@ -61,37 +61,73 @@ Scenario: User edits existing search
   And I click 'Save and continue'
   Then I am on the 'my cloud project' page
 
-Scenario: User exports and downloads results
```

Comment:
Let's add `@skip-staging` to these scenarios (the first 5 should be ok).

Reply:
Will do